src/reducers/wallet/NewWalletSetup.js

```diff
@@ -21,34 +21,22 @@ const INITIAL_STATE = {
  */
 
 export default (state = INITIAL_STATE, action) => {
-  // console.log(action);
   switch (action.type) {
     case actions.QRSCANNER_DATA:
-      console.log(action.payload);
       return { ...state, QrData: action.payload };
-    case actions.CREATING_NEW_WALLET:
-      console.log('wallet reducer: ');
-      console.log(action.payload);
+    case actions.CREATING_NEW_WALLET:      
       return { ...state, newWallet: true, wallet: action.payload };
     case actions.NEW_WALLET_NAME:
       return { ...state, walletName: action.payload };
     case actions.ADD_TOKEN_SETUP:
       var current = state.tokens;
```

Comment:
use let or const instead of var @CFDevelopment

Reply:
u need to let up lol

Reply:
fixed